boards/spracing/h7extreme/default.cmake

```diff
@@ -0,0 +1,121 @@
+
+px4_add_board(
+	PLATFORM nuttx
+	VENDOR spracing
+	MODEL h7extreme
+	LABEL default
+	TOOLCHAIN arm-none-eabi
+	ARCHITECTURE cortex-m7
+	ROMFSROOT px4fmu_common
+	SERIAL_PORTS
+#		GPS1:/dev/ttyS0
+#		RC:/dev/ttyS1
+#		TEL2:/dev/ttyS2
+#		TEL4:/dev/ttyS3
+	DRIVERS
+		adc
+		barometer # all available barometer drivers
+		#batt_smbus
+		camera_capture
+		camera_trigger
+		differential_pressure # all available differential pressure drivers
+		distance_sensor # all available distance sensor drivers
+		dshot
+		gps
+		#heater
+		#imu # all available imu drivers
+		#imu/adis16448
+		#imu/adis16477
+		#imu/adis16497
+		#imu/bmi088
+		imu/mpu6000
```

Comment:
Did you try `imu/invensense/icm20602` here?

Reply:
Not yet. I am struggling with free time. Soonest I find some time I'll test it.

Reply:
Strange, I see that you have it on in the defconfig. It's definitely working on the other H7 boards, as you can see it's obvious when it's not...

Reply:
I think the problem here SPI drivers can't be in ITCM because ITCM and DMA are not connected. 
I removed the SPI driver from ITCM and the CPU usage dropped from 45% to 20% for SPI. 
Now I have another problem when I am trying to calibrate gyro is working but the accelerometer is unresponsive, hm...

Reply:
The NuttX SPI driver has it's own DMA capable buffer (CONFIG_STM32H7_SPI1_DMA_BUFFER) that it uses for the large FIFO transfer (anything more than reading 1 or 2 registers) `CONFIG_STM32H7_SPI_DMATHRESHOLD`.

It sounds like it was working earlier, but just incredibly slow executing something relatively hot out of flash. We should try moving over the entire `icm20602` and dependencies.